book/kernel/scheduling.md

```diff
@@ -90,7 +90,7 @@ If the number of processes is lower than the number of cpu we don't need to swit
 If the number of processes is higher than the number of cpu we need to switch X process. Were X is equal to:
 
 ```c
-size_t schedule_count = MIN(cpu_count(), running - cpu_count());
```

Comment:
?

Reply:
Our MIN() macro is now name m_min() because everything in our math library is prefixed with m_ ;)

Reply:
fair